pkg/volume/cephfs/cephfs.go

```diff
@@ -407,6 +411,12 @@ func (cephfsVolume *cephfs) execFuseMount(mountpoint string) error {
 	mountArgs = append(mountArgs, cephfsVolume.path)
 	mountArgs = append(mountArgs, "--id")
 	mountArgs = append(mountArgs, cephfsVolume.id)
+	mountArgs = append(mountArgs, "--client_reconnect_stale")
+	mountArgs = append(mountArgs, "--client-quota")
+	mountArgs = append(mountArgs, "--admin_socket")
+	mountArgs = append(mountArgs, filepath.Join(configPath, fmt.Sprintf("ceph-fuse.%s.asok", cephfsVolume.id)))
```

Comment:
type: `asok` -> `sock`

Reply:
ceph use `asok` as socket file extension, I don't know why. But I think we can obey it.
http://docs.ceph.com/docs/master/rados/operations/monitoring/#using-the-admin-socket

Reply:
didn't know that, sure.